DependencyInjection/ZenstruckCacheExtension.php

```
@@ -30,7 +30,12 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
         $container->getDefinition('zenstruck_cache.crawler')->replaceArgument(0, new Reference('zenstruck_cache.client'));
 
         if ($mergedConfig['sitemap_provider']['enabled']) {
-            $container->setParameter('zenstruck_cache.sitemap_provider.host', $mergedConfig['sitemap_provider']['host']);
+
+            if (isset($mergedConfig['sitemap_provider']['host'])) {
+                $mergedConfig['sitemap_provider']['hosts'] = array_merge($mergedConfig['sitemap_provider']['hosts'], array($mergedConfig['sitemap_provider']['host']));
+            }
+
+            $container->setParameter('zenstruck_cache.sitemap_provider.hosts', $mergedConfig['sitemap_provider']['hosts']);
```

I think we should throw an exception is no hosts are set.